tests/Unit/Core/Language/Pack/Loader/RemoteLanguagePackLoaderTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Core\Language\Pack\Loader;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Foundation\Version;
+use PrestaShop\PrestaShop\Core\Language\Pack\Loader\RemoteLanguagePackLoader;
+
+final class RemoteLanguagePackLoaderTest extends TestCase
+{
+    public function testGetLanguagePackUrl()
+    {
+        $version = '8.0.0';
+        $basePath = 'http://mekey-i18n.com';
+        $locale = 'fr-FR';
+        $packLoader = new RemoteLanguagePackLoader(Version::buildFromString($version), $basePath);
+
+        $languagePackLocaleUrl = $packLoader->getLanguagePackUrl($locale);
+        $this->assertStringStartsWith($basePath, $languagePackLocaleUrl);
+        $this->assertStringEndsWith("{$version}/{$locale}/{$locale}.zip", $languagePackLocaleUrl);
+        $this->assertTrue(filter_var($languagePackLocaleUrl, FILTER_VALIDATE_URL) !== false, "Invalid URL found for {$languagePackLocaleUrl}");
+
+        $languagePackLangListUrl = $packLoader->getLanguagePackUrl();
+        $this->assertStringEndsWith("{$version}/available_languages.json", $languagePackLangListUrl);
+        $this->assertStringStartsWith($basePath, $languagePackLangListUrl);
+        $this->assertTrue(filter_var($languagePackLangListUrl, FILTER_VALIDATE_URL) !== false, "Invalid URL found for {$languagePackLangListUrl}");
```

Comment:
Can you please add a use case without the second parameter to ensure the default const is correctly used?